gulpfile.js

```diff
@@ -17,14 +17,17 @@ const config = {
 		cjs: 'build/modules/cjs',
 		esnext: 'build/modules/esnext'
 	},
-	DOCS_DIR: 'build/docs',
 	BUNDLE_DIR: 'build/dist',
 	WORKSPACE_DIR: '..',
 
 	// Path to the default configuration file for bundler.
 	BUNDLE_DEFAULT_CONFIG: 'dev/bundles/build-config-standard.js',
 
 	DOCUMENTATION: {
+		// Path to the temporary documentation files.
+		SOURCE_DIR: '.docs',
```

Comment:
I don't like this. The comment says something different than the name of this property. And do we have to keep some temporary stuff in the root dir? Can't we keep it somewhere in `build/`?